pkg/reconciler/v1alpha1/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -206,6 +221,37 @@ var allFinishedState = PipelineRunState{{
 	},
 }}
 
+var oneTaskFailedToRetry = PipelineRunState{{
+	PipelineTask: &pts[3],
+	TaskRunName:  "pipelinerun-mytask-failed",
+	TaskRun:      makeFailed(trs[0]),
+	ResolvedTaskResources: &resources.ResolvedTaskResources{
+		TaskSpec: &task.Spec,
+	},
+}, {
+	PipelineTask: &pts[0],
+	TaskRunName:  "pipelinerun-mytask1",
+	TaskRun:      nil,
+	ResolvedTaskResources: &resources.ResolvedTaskResources{
+		TaskSpec: &task.Spec,
+	},
+}}
+var oneTaskFailedWithRetry = PipelineRunState{{
+	PipelineTask: &pts[3],
+	TaskRunName:  "pipelinerun-mytask-failed",
+	TaskRun:      makeRetried(*makeFailed(trs[0])),
+	ResolvedTaskResources: &resources.ResolvedTaskResources{
+		TaskSpec: &task.Spec,
+	},
+}, {
+	PipelineTask: &pts[0],
+	TaskRunName:  "pipelinerun-mytask1",
+	TaskRun:      nil,
+	ResolvedTaskResources: &resources.ResolvedTaskResources{
+		TaskSpec: &task.Spec,
+	},
+}}
```

Comment:
it doesnt look like any tests cases are using these two? (also it looks like they might be the same)